### DISCUSSION TITLE
Removal of `http.cookies` / `http.cookiejar` from Windows installations

### DISCUSSION THREAD

**Discussion by Athanasius** [EDCD/EDMarketConnector#1125] [Plugin Development]
We're addressing https://github.com/EDCD/EDMarketConnector/issues/569 and I thought to check all of the plugins we list on https://github.com/EDCD/EDMarketConnector/wiki/Plugins#available-plugins---confirmed-working-under-python-37 - none of them appear to use these python modules.

If any EDMC plugin developer *does* make use of them please let us know here else you might find your plugin will stop working until you package those module(s) yourself.

**Edit: It turns out that core EDMC code use of `http` causes full inclusion of the module/submodules in library.zip anyway.**

**So, it *will* remain available, but only *because* of that core code use.  We never listed this on https://github.com/EDCD/EDMarketConnector/blob/main/PLUGINS.md#available-imports , and still won't.**

**Comment by aarronc** [comment]
we don't use this and don't plan on either :)

**Reply by seldonlabs** [reply]
I don't use it in my plugin, thanks for the warning. 👍

**Comment by vStone** [comment]
If plugins would use external libraries, instructions on how to build/package/distribute them would be handy (but not a requirement)

**Reply by Athanasius** [reply]
https://github.com/EDCD/EDMarketConnector/blob/main/PLUGINS.md#packaging-extra-modules

**Comment by Athanasius** [comment]
It turns out that core EDMC code use of `http` causes full inclusion of the module/submodules in library.zip anyway.

So, it *will* remain available, but only *because* of that core code use.  We never listed this on https://github.com/EDCD/EDMarketConnector/blob/main/PLUGINS.md#available-imports , and still won't.
